beacon_node/network/src/sync/block_lookups/mod.rs

```diff
@@ -743,14 +764,19 @@ impl<T: BeaconChainTypes> BlockLookups<T> {
         cx: &mut SyncNetworkContext<T>,
     ) {
         let Some(mut lookup) = self.single_block_lookups.remove(&target_id) else {
+            trace!(self.log, "Unknown single block lookup"; "target_id" => target_id);
             return;
         };
 
         let root = lookup.block_root();
         let request_state = R::request_state_mut(&mut lookup);
 
-        let Ok(peer_id) = request_state.get_state().processing_peer() else {
-            return;
+        let peer_id = match request_state.get_state().processing_peer() {
+            Ok(peer_id) => peer_id,
+            Err(e) => {
+                trace!(self.log, "Attempting to process single block lookup in bad state"; "id" => target_id, "response_type" => ?R::response_type(), "error" => e);
+                return;
+            }
```

Comment:
Added this two traces to debug tests, as they were super helpful. This cases should never happen in practice and if they do, important events are ignored. I think they should be at least debug, thoughts?

Reply:
Yes, debug sounds good to me